agent/cmd/registrator/main.go

```diff
@@ -25,6 +25,7 @@ func Main(args []string) {
 	apiAddr := cmd.String("api-addr", "api.buoyant.cloud:443", "address of the Buoyant Cloud API")
 	kubeConfigPath := cmd.String("kubeconfig", "", "path to kube config")
 	insecure := cmd.Bool("insecure", false, "disable TLS in development mode")
+	agentMetadataMap := cmd.String("agent-metadata-map", registrator.DefaultAgentMetadataConfigMap, "the name of the agent metadata map")
```

Comment:
i think it's ok to define `DefaultAgentMetadataConfigMap` as a private constant in this main, unless we're going to need it in the `registrator` package or elsewhere.